pkg/term/deprecated.go

```diff
@@ -0,0 +1,84 @@
+// Package term provides structures and helper functions to work with
+// terminal (state, sizes).
+//
+// Deprecated: use github.com/moby/term instead
+package term // import "github.com/docker/docker/pkg/term"
+
+import (
+	"github.com/moby/term"
+)
+
+// EscapeError is special error which returned by a TTY proxy reader's Read()
+// method in case its detach escape sequence is read.
+// Deprecated: use github.com/moby/term.EscapeError
+type EscapeError = term.EscapeError
+
+// State represents the state of the terminal.
+// Deprecated: use github.com/moby/term.State
+type State = term.State
+
+// Winsize represents the size of the terminal window.
+// Deprecated: use github.com/moby/term.Winsize
+type Winsize = term.Winsize
+
+var (
```

Comment:
If I remember correctly, for a group like this you can do

```go
// Deprecated: use github.com/moby/term instead
var (
...
)
```

and then you don't have to document/deprecate each individual one.

Not a big deal anyway